src/status_im/ui/screens/wallet/events.cljs

```diff
@@ -86,37 +86,39 @@
 
 (handlers/register-handler-fx
   :update-wallet
-  (fn [{{:keys [web3 accounts/current-account-id network] :as db} :db} [_ symbols]]
-    {:get-balance {:web3          web3
-                   :account-id    current-account-id
-                   :success-event :update-balance-success
-                   :error-event   :update-balance-fail}
-     :get-tokens-balance {:web3          web3
-                          :account-id    current-account-id
-                          :symbols       symbols
-                          :chain         (ethereum/network->chain-keyword network)
-                          :success-event :update-token-balance-success
-                          :error-event   :update-token-balance-fail}
-     :get-prices  {:from          "ETH"
-                   :to            "USD"
-                   :success-event :update-prices-success
-                   :error-event   :update-prices-fail}
-     :db          (-> db
-                      (clear-error-message :prices-update)
-                      (clear-error-message :balance-update)
-                      (assoc-in [:wallet :balance-loading?] true)
-                      (assoc :prices-loading? true))}))
+  (fn [{{:keys [web3 accounts/current-account-id network network-status] :as db} :db} [_ symbols]]
+    (when-not (= network-status :offline)
```

Comment:
not sure if this recommendation is applicable here but https://github.com/bbatsov/clojure-style-guide#not-equal